src/test/java/core/basesyntax/service/FileWriterImplTest.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertThrows;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class FileWriterImplTest {
+    private static final String FILE_TO = "src/main/resources/TestDataBase.csv";
```

Comment:
use test resources directory for the test files 
```suggestion
    private static final String FILE_TO = "src/test/resources/TestDataBase.csv";
```